src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/DryRunNotificationService.kt

```diff
@@ -77,9 +77,10 @@ class DryRunNotificationService(
     val cutoffTime = LocalDateTime.now(clock).minusMinutes(CUTOFF_MINUTES)
 
     val usersWithoutRecentActivity = details
+      .filter { it.quantumId != null && it.shiftModified != null }
```

Comment:
@andymarke I know I can use !! after this filtering. I noticed i could do a mapNotNull {} to a copy dto with non-nullable fields but not sure this is worth doing?
Also re the original code which does not have this filtering, I reasoned that there are no other code or logic changes so the values were always not  null.